packages/dropdowns/examples/tree-menu.md

```diff
@@ -65,7 +65,7 @@ const renderItems = () => {
   <Trigger>
     <Button>Tree Layout</Button>
   </Trigger>
-  <Menu placement="end" hasArrow style={{ width: 200, height: 258 }}>
+  <Menu placement="end" hasArrow>
```

Comment:
Why is this removed? Without a fixed height, the size of the tree will change based on the number of items.

Reply:
When the padding on the list items was revised it resulted in a bunch of empty space at the bottom on the menu. I was going to try to do the math to figure out the new menu height but it didn't seem like a very good practice from a responsive standpoint to hard code widths and heights in the examples we provide.

In general glad you called this out, had a question: is a reason why we are choosing to replace the menu in nested navigation vs showing a new one? Something like [this example](https://share.getcloudapp.com/z8uxA492)? That's what I have typically seen for this behavior and it would eliminate this height jumping.

Reply:
That pattern only works on desktop where there is enough space to nest relative to the other elements. This was the design decision chosen at the time of development.

> I was going to try to do the math to figure out the new menu height but it didn't seem like a very good practice

With the ability for the menu content to overflow this is still responsive. Since this is a separate design decision I'm not sure we should be adding this here.

Reply:
I'm unsure who to defer to here. I would advocate for not having explicit heights inline in the example if we don't have to but if design has traditionally said to do that then I guess I need to file a Jira ticket for it and appeal to someone?

Reply:
@rossmoody, I'll work together with you on this one to help get it right.